specification/redis/resource-manager/Microsoft.Cache/stable/2021-06-01/redis.json

```diff
@@ -2812,8 +2812,7 @@
         {
           "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationStatusResult"
         }
-      ],
-      "x-ms-external": true
```

Comment:
Is there any reason to remove this?

Reply:
@akning-ms its inclusion is creating problems when generating our .NET SDK. It seems like it should have never been set in the first place.

Reply:
as it seems caused GO & JS breaking change. @tadelesh @qiaozha to confirm

Reply:
The Go breaking is from code generator and could be ignored.

Reply:
@qiaozha is the JS breaking change OK also?